src/databricks/labs/ucx/workspace_access/generic.py

```diff
@@ -110,21 +109,19 @@ def _response_to_request(
 
     @rate_limited(max_requests=30)
     def _applier_task(self, object_type: str, object_id: str, acl: list[iam.AccessControlRequest]):
-        for _i in range(0, 3):
```

Comment:
So this used a loop to apply the same (full set, mind you) permissions until they are all available on the object. There is no error handling here (that is, no `try/except`).